FAQ/en.md

```diff
@@ -0,0 +1,61 @@
+# Frequently Asked Questions
+
+## General
+
+### What state is Quaver currently in?
```

Comment:
Rename this question to `How far in development is Quaver?`

Reply:
Done